### PR TITLE
Update release notes for MakeItNative 1.2.0 release

### PR DESCRIPTION
**Pull request by smarki**

### CODE REVIEW

File: content/releasenotes/mobile/make-it-native-app.md [hunk 1]
```diff
@@ -11,6 +11,28 @@ tags: ["Native", "Android", "iOS", "Apple", "Android"]
...
+
+On iOS the QR Code Scanner performs more consistently. 
+
+On iOS the status bar contrasts with the default Atlas UI theme.
+
+The Android app was re-architectured to fix various performance bottlenecks.
+
+### Known Issues
+
+* When using the `Touchable` option in the inspector overlay, the Mendix Native app crashes.
```
Comment (ConnorLand) [inline content/releasenotes/mobile/make-it-native-app.md]: @smarki can you please slack me the JIRA issue of this KI?

Comment (smarki) [reply content/releasenotes/mobile/make-it-native-app.md]: https://mendix.atlassian.net/browse/UICORE-417?atlOrigin=eyJpIjoiMjhiMDkwNmVlNTZmNDI1ZWFjYzZhNjk3ZTIxNjc2NzYiLCJwIjoiaiJ9

Comment (smarki) [reply content/releasenotes/mobile/make-it-native-app.md]: UICORE-417

Comment (ConnorLand) [reply content/releasenotes/mobile/make-it-native-app.md]: Awesome thank you.

**Pull request merged**